test/cli/files.js

```diff
@@ -127,6 +127,23 @@ describe('files', () => runOnAndOff((thing) => {
       })
   })
 
+  it('add with piped argument', () => {
+    // echo 'src/init-files/init-docs/readme' | jsipfs files add
+    return ipfs('files add', { piped: 'src/init-files/init-docs/readme' })
```

Comment:
If I understood it correctly, it seems that it might be an issue with the CLI itself. The piped arguments work exactly the same way as if the arguments were passed directly:

```
⌁46% [home:~/dev/js-ipfs] cli_pipe_474(+1/-2)+ 2s 1 ± node src/cli/bin.js files add  $(head -c 10 /dev/urandom  | base64)
```

```
/Users/home/dev/js-ipfs/node_modules/yargs/yargs.js:1079
      else throw err
           ^

Error: ENOENT: no such file or directory, stat 're+g0ZVkYUUsKg=='
    at Object.fs.statSync (fs.js:967:11)
    at checkPath (/Users/home/dev/js-ipfs/src/cli/commands/files/add.js:32:10)
    at Object.handler (/Users/home/dev/js-ipfs/src/cli/commands/files/add.js:72:20)
    at Object.self.runCommand (/Users/home/dev/js-ipfs/node_modules/yargs/lib/command.js:233:22)
    at Object.Yargs.self._parseArgs (/Users/home/dev/js-ipfs/node_modules/yargs/yargs.js:990:30)
    at Object.self.runCommand (/Users/home/dev/js-ipfs/node_modules/yargs/lib/command.js:204:45)
    at Object.Yargs.self._parseArgs (/Users/home/dev/js-ipfs/node_modules/yargs/yargs.js:990:30)
    at Object.Yargs.self.parse (/Users/home/dev/js-ipfs/node_modules/yargs/yargs.js:532:23)
    at utils.getIPFS (/Users/home/dev/js-ipfs/src/cli/bin.js:67:6)
    at IPFS.node.once (/Users/home/dev/js-ipfs/src/cli/utils.js:64:5)
```

Or am I missing something?